src/typing/typeloadCheck.ml

```diff
@@ -346,7 +347,7 @@ module Inheritance = struct
 			try
 				let t2, f2 = class_field_no_interf c i in
 				let t2, f2 =
-					if ctx.com.config.pf_overload && (f2.cf_overloads <> [] || has_class_field_flag f2 CfOverload) then
+					if f2.cf_overloads <> [] || has_class_field_flag f2 CfOverload then
```

Comment:
Shouldn't this also check for class being extern?

Reply:
I don't think that's necessary, it would mean that we have bad state because neither `cf_overloads` nor the `CfOverload` flag should be set.